scripts/twister

```diff
@@ -1322,5 +1327,6 @@ if __name__ == "__main__":
     try:
         main()
     finally:
-        if os.isatty(1): # stdout is interactive
+        if os.isatty(1) and (os.name != "nt"):
```

Comment:
Why not invert the order of that `if` statement so that we don't make the `isatty()` call if the OS is `nt`?

Reply:
Sure, it make more sense :) I've introduced your suggestion